src/breadcrumbs/main.cjs

```diff
@@ -0,0 +1,62 @@
+/**
+ * Blockquote component
+ * Common js module to support DXP Component Service.
+ */
+module.exports = async function (input, info) {
+  const getBreadcrumbData = function () {
+    let breadcrumbHtml = "";
+    const navData = [
```

Comment:
isn't navData meant to be an input on the json or pulled in via some other direction?